app/models/good_job/base_execution.rb

```diff
@@ -5,6 +5,9 @@ module GoodJob
   # which both read out of the same table.
   class BaseExecution < BaseRecord
     include ErrorEvents
+    include Filterable
+    include Lockable
+    include Reportable
```

Comment:
Would love if I could group this; `include Filterable, Lockable, Reportable`
but rubocop is not allowing it :)

Reply:
Seems like a standard style; Rails uses it too. My digging surfaced this, but not the best explanation :-) 

https://github.com/rubocop/ruby-style-guide/pull/622

Reply:
I agree definitely not worth changing this rule, it just looked nicer in this particular situation to me :)

I think this rule is on by default because of: 
1. Git diffing
2. Error tracing - line number, if there are multiple mixins on 1 line could be harder to trace